services/docker/docker-size.service.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const prettyBytes = require('pretty-bytes')
+const { anyInteger } = require('../validators')
+const { buildDockerUrl, getDockerHubUser } = require('./docker-helpers')
+const { BaseJsonService } = require('..')
+const { NotFound } = require('..')
+
+const buildSchema = Joi.object({
+  name: Joi.string(),
+  full_size: anyInteger,
+}).required()
+
+module.exports = class DockerSize extends BaseJsonService {
+  static get category() {
+    return 'size'
+  }
+
+  static get route() {
+    return buildDockerUrl('image-size', true)
```

Comment:
So in the absence of the user providing a value for the optional tag param, which tag are we using for the size value?

Reply:
Per: https://github.com/badges/shields/blob/88721950f028d78cc4c0be278f6b80d8a5e0479e/services/docker/docker-size.service.js#L61 we use `latest` when a tag is not provided, both for the size and version abdges.